src/data/tests/ContentHighlightsTestData/index.js

```diff
@@ -0,0 +1,18 @@
+import { testCourseHighlightsData, testCourseData } from './utils';
+import { initialStateValue, ContentHighlightsContext } from './context';
+
+const rawTestCourseData = require('./testCourseData.json');
+const rawTestCourseHighlights = require('./testCourseHighlights.json');
+const testCourseAggregation = require('./testCourseAggregation.json');
+const testHighlightSet = require('./testHighlightSet.json');
```

Comment:
similar to another comment, might be worth looking into `rosie` or similar to generate JS objects using a factory, which would likely be a bit more maintainable/scalable/customizable.